tests/core/tools/absoluterectposition.js

```diff
@@ -3,12 +3,23 @@
 ( function() {
 	'use strict';
 
-	var tests = {
+	var scrollPositionMock,
+		windowMock,
+		tests = {
 		setUp: function() {
+			var window = CKEDITOR.document.getWindow();
+
+			windowMock = sinon.stub( CKEDITOR.document, 'getWindow' );
+			windowMock.returns( window );
+
 			CKEDITOR.document.getBody().setStyle( 'height', '5000px' );
+
+			scrollPositionMock = sinon.stub( window, 'getScrollPosition' );
```

Comment:
You could simplify it by stubbing getScrollPosition directly in our window wrapper's prototype, like so:

> scrollPositionMock = sinon.stub( CKEDITOR.dom.window.prototype, 'getScrollPosition' );